setup.py

```diff
@@ -2,7 +2,6 @@
 
 
 install_requires = [
-    "Django>=1.11,<2.3",
```

Comment:
I think when @willbarton made the suggestion that led to this PR, he was probably mostly or only talking about removing this line.

We probably want to leave in some or all of the stuff that tests various Django versions, to make sure that we don't put code in this project that is compatible with earlier versions of Django that are still supported by the versions of Wagtail _this project_ supports.

What do you think, @willbarton?

Reply:
@willbarton Wagtail 2.9 was released today, Do we want to start thinking about heading right to that instead of 2.8?

Reply:
Changes made as requested